framework/src/userobject/SideUserObject.C

```diff
@@ -46,3 +46,49 @@ SideUserObject::SideUserObject(const InputParameters & parameters)
   for (const auto & var : coupled_vars)
     addMooseVariableDependency(var);
 }
+
+void
+SideUserObject::getFaceInfos()
+{
+  _face_infos.clear();
+
+  // Either the element or the (active) neighbor is a valid argument to get a face info
+  const Elem * equal_level_neighbor = _current_elem->neighbor_ptr(_current_side);
```

Comment:
I realize now that this may not be an equal level neighbor, so let's change the name of this local variable. Maybe `side_neighbor`?